protocol-units/execution/aptos/opt-executor/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "monza-opt-executor"
+name = "movement-opt-executor"
```

Comment:
I would call this `aptos-opt-executor`

Reply:
This would make it look like one of the aptos-core crates, which would be confusing.

Reply:
How about `suzuka-opt-executor`?

Reply:
This is now shared between Monza and Suzuka, though I don't know if it will always be so.